src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -197,6 +197,19 @@ configurationRegistry.registerConfiguration({
 	}
 });
 
+configurationRegistry.registerConfiguration({
+	'id': 'notebook',
+	'title': 'Notebook',
+	'type': 'object',
+	'properties': {
+		'notebook.showAllKernels': {
+			'type': 'boolean',
+			'default': false,
+			'description': localize('notebook.showAllKernels', "(Experimental) show all kernels for the current notebook provider.")
```

Comment:
Should be preview instead to keep consistent with our other "early" features

Reply:
Also this should be under the preview flag - in which case is this option even really necessary? Not sure if there's a case for someone wanting preview features enabled but not wanting to see these.

Reply:
I really do think that we shouldn't turn this on for everyone. For example, by default, users will see 2 Python kernels with Anaconda (including one that has a crazy string as the display name), and so I really think we should roll this out slowly and methodically, until we have more data.

Totally hear you on the Experimental vs Preview thing. I'll make that change.

Reply:
Alright. This still should be controlled by the preview features flag as well though